templates/mustache-templates/teamsBot.ts.mustache

```diff
@@ -2,42 +2,52 @@
 import { default as axios } from "axios";
 import * as querystring from "querystring";
 {{/IS_ME}}
-import { TeamsActivityHandler, CardFactory, TurnContext{{#IS_BOT}}, SigninStateVerificationQuery, ActionTypes, BotState{{/IS_BOT}}} from "botbuilder";
-{{#IS_BOT}}import { MainDialog } from "./dialogs/mainDialog";{{/IS_BOT}}
+import { TeamsActivityHandler, CardFactory, TurnContext} from "botbuilder";
+{{#IS_BOT}}
+const rawWelcomeCard = require("./adaptiveCards/welcome.json");
+const ACData = require("adaptivecards-templating");
+{{/IS_BOT}}
 
 export class {{#IS_BOT}}TeamsBot{{/IS_BOT}}{{^IS_BOT}}MessageExtensionBot{{/IS_BOT}} extends TeamsActivityHandler {
   {{#IS_BOT}}
-  conversationState: BotState;
-  userState: BotState;
-  dialog: MainDialog;
-  dialogState: any;
-
-  /**
-   * @param {ConversationState} conversationState
-   * @param {UserState} userState
-   * @param {Dialog} dialog
-   */
-  constructor(conversationState: BotState, userState: BotState, dialog: MainDialog) {
+  // record the likeCount
+  likeCountObj: { likeCount: number };
+
+  constructor() {
     super();
-    if (!conversationState) {
-      throw new Error("[TeamsBot]: Missing parameter. conversationState is required");
-    }
-    if (!userState) {
-      throw new Error("[TeamsBot]: Missing parameter. userState is required");
-    }
-    if (!dialog) {
-      throw new Error("[TeamsBot]: Missing parameter. dialog is required");
-    }
-    this.conversationState = conversationState;
-    this.userState = userState;
-    this.dialog = dialog;
-    this.dialogState = this.conversationState.createProperty("DialogState");
+
+    this.likeCountObj = { likeCount: 0 };
 
     this.onMessage(async (context, next) => {
       console.log("Running dialog with Message Activity.");
```

Comment:
I know this isn't in your PR - but can you clean up this message? It's no longer a "dialog" since we are removing the dialog support. Maybe `"Received message: " + context.activity.text` or something.

Reply:
OK，thank you!

Reply:
removed "dialog", change it to "Running with Message Activity"